tilequeue/query/fixture.py

```diff
@@ -0,0 +1,56 @@
+from collections import namedtuple
+from shapely.geometry import box
+
+
+LayerInfo = namedtuple('LayerInfo', 'min_zoom_fn props_fn')
+
+
+class DataFetcher(object):
+
+    def __init__(self, layers, rows):
+        """
+        Expect layers to be a dict of layer name to LayerInfo. Expect rows to
+        be a list of (fid, shape, properties).
+        """
+
+        self.layers = layers
+        self.rows = rows
+
+    def __call__(self, zoom, unpadded_bounds):
+        read_rows = []
+        bbox = box(*unpadded_bounds)
+
+        for (fid, shape, props) in self.rows:
+            # reject any feature which doesn't intersect the given bounds
+            if bbox.disjoint(shape):
+                continue
+
+            # place for assembing the read row as if from postgres
+            read_row = {}
+
+            for layer_name, info in self.layers.items():
+                # TODO: what should meta be?
```

Comment:
Added as #230. This issue is somewhat nebulous at this point, but I suspect that it'll become clearer as we try to implement more parts of the system.